### PR TITLE
Improved README and identifier refactoring

### PR DESCRIPTION
**Pull request by mshadbolt**
Relates to [DIG-1331](https://candig.atlassian.net/browse/DIG-1331), [DIG-1356](https://candig.atlassian.net/browse/DIG-1356), [DIG-1357](https://candig.atlassian.net/browse/DIG-1357)

- Restructure, correct and add details to the main README file
- refactor identifiers for clarity
  - `genomic_id` -> `genomic_file_id` 
  -  `genomic_sample_id` - > `genomic_file_sample_id`

@daisieh and @OrdiNeu to review code changes

Would be great to have @yavyx 's input on the [README](https://github.com/CanDIG/candigv2-ingest/blob/mshadbolt/update_docs_refactor_sample_id/README.md). (as it is very restructured it will be easier to read on the [main page](https://github.com/CanDIG/candigv2-ingest/blob/mshadbolt/update_docs_refactor_sample_id/README.md) instead of looking at the changes)

[DIG-1331]: https://candig.atlassian.net/browse/DIG-1331?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DIG-1356]: https://candig.atlassian.net/browse/DIG-1356?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DIG-1357]: https://candig.atlassian.net/browse/DIG-1357?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by yavyx** [comment]
Readme looks great! The only issue is confirming that we can ingest `.cram` and `.cram.crami` files.

File: README.md [hunk 1]
```diff
@@ -39,73 +41,68 @@ source env.sh
...
-Restart the OPA container to take effect
+The clinical ingest API runs at `/ingest/clinical`. Simply send a request with an authorized bearer token and a JSON body with your `DonorWithClinicalData` object. See the swagger UI/[schema](ingest_openapi.yaml) for the response format.
 
-## Ingest clinical data
-Note: To upload a DonorWithClinicalData object, you should produce one using the [CanDIG ETL](https://github.com/CanDIG/clinical_ETL_code), then store its result in a JSON file under the key "donors". See single_ingest.json for an example.
+#### Command line
 
-### Command line
-Before you can ingest the clinical data, you need to format your data into the json ingest format using the [clinical_ETL](https://github.com/CanDIG/clinical_ETL_data) and put it in the katsu `data` folder, then set the environment variable `CLINICAL_DATA_LOCATION`:
+This method is mainly used for development work.
```
Comment (daisieh) [inline README.md]: For now, this is also used for ingest of jsons that are too big to send easily via POST.

**Review by daisieh — APPROVED** [review]
Looks good! I made one tiny comment.

**Comment by mshadbolt** [comment]
Added @daisieh 's suggestion and she has also confirmed that pysam can handle cram and cram.crai so this also resolve's @yavyx 's concern. merging.

**Pull request merged**